### DISCUSSION TITLE
文档中 “frpc 基本使用指南” 章节的小问题

### DISCUSSION THREAD

**Discussion by LTHPKBTE** [natfrp/wiki#269] [想法] [answered]
标题指的是[frpc 基本使用指南](https://doc.natfrp.com/frpc/usage.html) 这个页面。
目前这个页面中不同系统的安装操作被分成多个选项卡，但是不同选项卡并没有各自的链接（类似 usage.html#running-frpc 这样的章节链接），复制给新人使用时无法直接到达相关系统页面，多个链接到使用指南中 ”启动隧道“ 章节的链接也同时失效。

[a comment by an automated account is not shown]

**Reply by SanaeAkahane** [reply]
现场p图做表情包了属于是
